frameworks/security.json

```
@@ -176,6 +176,12 @@
                 "name": "Anonymous access enabled"
             }
         },
+        {
+            "controlID": "C-0264",
+            "patch": {
+            "name": "Data in rest encryption - Persistent Volumes are encrypted (CC1.1,CC6.7)"
```

```suggestion
            "name": "PersistentVolume without encyption"
```
let's use the original control name, this is the name from soc2 fw